src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx

```diff
@@ -200,6 +229,46 @@ export const SpaceEditFormNG: React.FC<SpaceEditFormNGProps> = ({
         </AdminSidebarSubTitle>
         <AdminSpacesListItem title="The basics" isOpened>
           <>
+            <AdminSection title="Rename your space" withLabel>
+              <AdminInput
+                name="name"
+                placeholder="Space Name"
+                register={register}
+                errors={errors}
+                required
+              />
+            </AdminSection>
+            <AdminSection title="Subtitle" withLabel>
+              <AdminInput
+                name="subtitle"
+                placeholder="Subtitle for your space"
+                register={register}
+                errors={errors}
+              />
+            </AdminSection>
+            <AdminSection title="Description" withLabel>
+              <AdminInput
+                name="description"
+                placeholder={`Let your guests know what they’ll find when they join your space. Keep it short & sweet, around 2-3 sentences maximum. Be sure to indicate any expectations for their participation.`}
+                register={register}
+                errors={errors}
+                height="120"
+                isTextarea
```

Comment:
`isTextarea`? No, it isn't. But https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/AdminTextarea/AdminTextarea.tsx is

Reply:
done da7ba06